doc/admin-guide/plugins/ts_lua.en.rst

```diff
@@ -131,6 +131,22 @@ is always available within lua script. This package can be introduced into Lua l
 
 `TOP <#ts-lua-plugin>`_
 
+ts.process_id
+-------------
+**syntax:** *val = ts.process_id()*
```

Comment:
Suggest you call this ``uuid`` rather than ``id`` since the latter implies numeric.